ui/react-components/components/AppointmentEditor/AppointmentEditor.jsx

```diff
@@ -54,6 +83,11 @@ const AppointmentEditor = props => {
         return false;
     };
```

Comment:
Remove appointmentDateProps - unused. It is in this file

Reply:
removed